git_machete/cli.py

```diff
@@ -558,11 +565,18 @@ def launch(orig_args: List[str]) -> None:
                     machete_client.annotate(branch, parsed_cli.annotation_text)
                 else:
                     machete_client.print_annotation(branch)
+        elif cmd == "clean":
+            machete_client.read_definition_file(perform_interactive_slide_out=should_perform_interactive_slide_out)
+            machete_client.delete_unmanaged(opt_yes=cli_opts.opt_yes)
```

Comment:
Hmmm I can see one case where the current ordering (delete unmanaged BEFORE checkout PRs) will be quite awkward: the case of a **brand new user** (empty .git/machete, some PRs opened on GitHub) who decided to run `git machete github sync` after reading README.

What's gonna happen is that `github sync` will suggest deleting his local branches (for which he's got opened PRs!), only to later re-checkout them in `checkout_github_prs`. This will be very confusing :/

How about reordering the operations once more both here and in `clean`?
1. checkout GH PRs
2. delete unmanaged
3. delete untracked